### DISCUSSION TITLE
Getting always error with ansible install

### DISCUSSION THREAD

**Discussion by Bleala** [mKeRix/room-assistant#851] [Q&A]
Hello guys,

i'm always getting the following error, when installing Room-Assistant with the ansible playbook

`fatal: [192.168.188.78]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'dict object' has no attribute 'path'\n\nThe error appears to be in '/root/ansible/room-assistant/roles/room-assistant/tasks/main.yml': line 38, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n  register: find_node\n- name: Set node capabilities\n  ^ here\n"}`

Does anyone know what to do here?

Greetings

**Comment by mKeRix** [comment]
For future reference, let's talk about this in mKeRix/ansible-playbooks#14
